app/views/home/home.html.erb

```diff
@@ -83,6 +92,34 @@
   .activity-dropdown {
     display: none;
   }
+
+  #questions-link-container {
+    padding: 94px 280px;
+    text-align: center;
+  }
+
+  #questions-link-container>h1 {
```

Comment:
This is looking good, but I'm not sure we absolutely need the extra formatting here. I wonder if we could remove the styles below this line... and maybe use `<hr />` for spacing? I guess I'm reluctant to build out a big style section just for this one feature. What do you think?